appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerInstance.kt

```diff
@@ -28,16 +28,16 @@ class AppsFlyerInstance(
         configSettings: Map<String, Any>?
     ) {
         configSettings?.let { settings ->
-            if (settings.containsKey(Config.MIN_TIME_BETWEEN_SESSIONS)) {
-                setMinsBetweenSessions(settings[Config.MIN_TIME_BETWEEN_SESSIONS] as Int)
+            if (settings.containsKey(Settings.TIME_BETWEEN_SESSIONS)) {
+                setMinsBetweenSessions(settings[Settings.TIME_BETWEEN_SESSIONS] as Int)
```

Comment:
not specifically part of this PR, but we should change the `as Int` to something safer, as that's going to throw if the value contained in it is not an Int.
Probably better to fail gracefully and still allow the SDK to get initialized further down - looks like iOS does this also.